components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -1007,22 +1012,18 @@ def validate_aws_config_dir(self):
                 "aws_config_directory should not be set when profile authentication is not used"
             )
 
-    def load_execution_container_name(self):
-        if self.execution_container is not None:
+    def load_container_image_ref(self):
+        if self.container_image_ref is not None:
             # Accept configured value for debug purposes
             return
 
-        os_release = dotenv_values(self._os_release_file_path)
```

Comment:
we should have removed the `dotenv_values` import and related dependency installation in this PR. @coderabbitai file an issue